### ISSUE TITLE
Sports site m3u support request 

### ISSUE THREAD

**Issue by kolekan** [dtankdempse/topembed-m3u#2]
Hi nice work with all those m3u playlists. I would like to ask, if possible, to create an m3u-repo for "abbasport.site" as well. Thanks for your work so far.

**Comment by dtankdempse** [comment]
Yes, the site seems to have the same layout, so it should be fairly easy.

**Comment by sp0oky16** [comment]
when will this be added

**Comment by dtankdempse** [comment]
Give me a couple of days.

**Comment by dtankdempse** [comment]
This has been added here: https://github.com/dtankdempse/abbasport-m3u
